### PR TITLE
add loop chain

### PR DESCRIPTION
**Pull request by PFC-developer**
thx.

### CODE REVIEW

File: loop/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        {
+          "denom": "token",
+          "exponent": 0
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "token",
+      "name": "Token",
+      "display": "token",
+      "symbol": "Token",
```
Comment (JeremyParish69) [inline loop/assetlist.json:17]: Please make symbol all CAPS

File: loop/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/loop/images/loop.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/loop/images/loop.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/loop/images/loop.png",
```
Comment (JeremyParish69) [inline loop/assetlist.json:24]: Using the same image for TOKEN and POA is confusing

Comment (JeremyParish69) [inline loop/assetlist.json:24]: No png has been uploaded

File: loop/chain.json [hunk 3]
```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "loop",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.loop.fans/",
+  "pretty_name": "Loop",
+  "chain_id": "loop-1",
+  "bech32_prefix": "loop",
```
Comment (JeremyParish69) [inline loop/chain.json:9]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: loop/chain.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        "low_gas_price": 0.0,
+        "average_gas_price": 0.0,
+        "high_gas_price": 0.0
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uupoa"
```
Comment (JeremyParish69) [inline loop/chain.json:30]: typo (an extra 'u')

**Comment by PFC-developer** [comment]
prefix is in process of being registered - https://github.com/satoshilabs/slips/pull/1709

File: loop/chain.json (hunk 3 above)
Comment (PFC-developer) [reply loop/chain.json:9]: https://github.com/satoshilabs/slips/pull/1709

**Comment by PFC-developer** [comment]
prefix is merged
https://github.com/satoshilabs/slips/commit/7bfd144a11b8a70503a1f5f2c8346c30b31e0b38

**Pull request merged**